### PR TITLE
chore: Redesign Dashboard

### PR DESCRIPTION
**Pull request by peterpaints**
#### What does this PR do?
- The dashboard has been redone to resemble Humphrey's designs as closely as possible.
#### Where should the reviewer start?
- Go to /dashboard in localhost
#### Screenshots (if appropriate)
screen shot 2018-06-08 at 00 43 46
screen shot 2018-06-08 at 00 44 14

### CODE REVIEW

**Comment by kimobrian** [comment]
@peterpaints Mind fixing those Circle CI issues

**Comment by peterpaints** [comment]
@kimobrian On it, just noticed tests are failing.

**Comment by peterpaints** [comment]
@kimobrian Tests Fixed.

**Comment by NaiRobley** [comment]
@peterpaints the UI looks good

**Comment by peterpaints** [comment]
Thanks @NaiRobley

**Comment by kimobrian** [comment]
@jimmykimani Can you provide him with the specifics. Font families, colours etc. All those small details

**Comment by jimmykimani** [comment]
@kimobrian Yes and also for the Icons

**Comment by kimobrian** [comment]
@jimmykimani & @peterpaints , I just added a `Work in Progress` label. Always do that when you are still working on the feature or implementing changes from PR reviews. There's a READY label that you should add when you are done with all the required changes.

**Comment by jimmykimani** [comment]
Noted @kimobrian

**Comment by jimmykimani** [comment]
@peterpaints Align the Incident title with the Incidents card

File: src/Components/IncidentFilter/IncidentFilter.scss [hunk 1]
```diff
@@ -55,6 +55,7 @@
       color: #000000;
       margin-right: auto;
       margin-left: 2rem;
+      font-family: 'DIN Pro Light';
     }
 
     .duration-filter {
```
Comment (jimmykimani) [inline src/Components/IncidentFilter/IncidentFilter.scss]: Add a cursor property to pointer on hover

**Pull request merged**